lib/basedriver/commands/find.js

```diff
@@ -64,18 +63,40 @@ commands.findElementsFromElement = async function (strategy, selector, elementId
   return await this.findElOrElsWithProcessing(strategy, selector, true, elementId);
 };
 
-helpers.findByImage = async function (b64Template, {checkingStaleness = false} = {}) {
-  let threshold = DEFAULT_MATCH_THRESHOLD;
-  if (this.settings) {
-    threshold = this.settings.getSettings().imageMatchThreshold;
-  }
+/**
+ * @typedef {Object} FindByImageOptions
+ * @property {boolean} shouldCheckStaleness - whether this call to find an
```

Comment:
can we also mention the default values for these properties or if any of them is required to set?